server/handlers/error.go

```diff
@@ -200,7 +200,7 @@ func ErrPatternFile(err error) error {
 }
 
 func ErrInvalidPattern(err error) error {
-	return errors.New(ErrInvalidPatternCode, errors.Alert, []string{"Invalid design, execution is infeasible", err.Error()}, []string{err.Error()}, []string{"Trait Definition is invalid or unable to process"}, []string{"Check Trait Definition"})
+	return errors.New(ErrInvalidPatternCode, errors.Alert, []string{"Invalid design, execution is infeasible", err.Error()}, []string{err.Error()}, []string{"Services/Traits definition is invalid or unable to process"}, []string{"Ensure that Services/Traits definition present in meshery design", "Ensure that Services/Traits definition is not invalid"})
```

Comment:
@Yashsharma1911 thanks. We don't use Traits any longer and will hopefully get rid of Services, too. @MUzairS15

Reply:
Removing traits from suggestion

Reply:
This has resurfaced recently, @MUzairS15